libraries/helpers.rb

```diff
@@ -127,7 +127,7 @@ def add_config(product, content)
       # foodcritic thinks we are accessing a node attribute
       node.run_state[:ingredient_config_data] ||= {}              # ~FC001
       node.run_state[:ingredient_config_data][product] ||= ''     # ~FC001
-      node.run_state[:ingredient_config_data][product] += content # ~FC001
+      node.run_state[:ingredient_config_data][product] += content unless node.run_state[:ingredient_config_data][product].include?(content) # ~FC001
```

Comment:
I read this as we only allow recipes to set the content data for a product once during a client run.  Would it be better to raise an exception (or at least a warning) so it doesn't silently bypass?